### PR TITLE
feature - update readme

### PR DESCRIPTION
**Pull request by at-quangphu**

### CODE REVIEW

**Comment by at-quangphu** [comment]
@at-daonguyen Please, help me review it. Many thanks. 😄

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 [![Build Status](https://travis-ci.org/tsrnd/realm-mapper-ios.svg?branch=master)](https://travis-ci.org/tsrnd/realm-mapper-ios)
-[![CocoaPods Compatible](https://img.shields.io/cocoapods/v/RealmMapper.svg)](https://img.shields.io/cocoapods/v/RealmMapper.svg)
+[![CocoaPods Compatible](https://img.shields.io/badge/pod-v3.0-blue.svg)](https://img.shields.io/badge/pod-v3.0-blue.svg)
```
Comment (at-daonguyen) [inline README.md]: Please revert this change.

**Comment by at-daonguyen** [comment]
#14

**Comment by at-quangphu** [comment]
@at-daonguyen Please, help me review it again. Thank you 😄
